docs/plugins/backup.md

```diff
@@ -46,7 +46,7 @@ plugin(
     b.bucket = 'acquia-cloud-database-test'
     b.files = [
```

Comment:
Yeah, my preference is always for *simple*. I can't think of a use case for asserting the permissions within the tarball.